src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawFigure();
```

Comment:
bad method name

this interface doesn't know anything about figures

you should use simple `draw()`